locations/alpineskyline.json

```diff
@@ -248,11 +248,17 @@
           {
             "name": "Entrance"
           },
+          {
```

Comment:
Time trial wasn't added as a check from what I can see
![image](https://github.com/hoppel16/ahit-poptracker/assets/38697343/4ce9ec9e-8fcf-42fe-9543-47e22e66c47b)

Reply:
scratch that, it was renamed to inside pon cluster since the time trial isnt actually required to get the item and its just spawned in by default